python/ray/_private/memory_monitor.py

```diff
@@ -38,9 +38,15 @@ def get_message(used_gb, total_gb, threshold):
         pids = psutil.pids()
         proc_stats = []
         for pid in pids:
-            proc = psutil.Process(pid)
-            proc_stats.append((get_rss(proc.memory_info()), pid,
-                               proc.cmdline()))
+            try:
+                proc = psutil.Process(pid)
+                proc_stats.append((get_rss(proc.memory_info()), pid,
```

Comment:
Would `proc.memory_info()` throw an exception?

Reply:
`proc.memory_info()` reads `/proc/pid/statm` file and `pro.cmdline` reads `/proc/pid/cmdline`. 
They will throw `NoSuchFile` exception instead of `NoSuchProcess`.  So the change is not correct.

It's hard to construct a unit test to cover it. Do you have more thoughts?

Reply:
How about we catch all kinds of exceptions here? I don't have an idea for unit testing neither.

Reply:
I guess in Windows the exception is not 'NoSuchFile' because not all things are files in Windows.

Reply:
I think the current solutions looks fine, and we can add another exception check once they are discovered. Thought?

Reply:
I prefer to catch the base exception here, in case there are other kinds of possible errors.

Reply:
@kfstorm @rkooo567 
After looking into the implementations  of `psutil.Process` and `psutil.memory_info`,  found that it wraps other exceptions as `NoSuchProcess`. It's unnecessary to catch other exceptions any more.

Reply:
@jovany-wang OK. Let's keep the current catch solution.